### ISSUE TITLE
Direct upload of final submission to OSM, attributed to user

### ISSUE THREAD

**Issue by spwoodcock** [hotosm/fmtm#1550]
**Is your feature request related to a problem? Please describe.**

- Following on from https://github.com/hotosm/fmtm/issues/1549
- The user should be able to submit their data once they have manually verified the submission.

**Describe the solution you'd like**

- An endpoint should be made for uploading the final submission GeoJSON directly to OSM.
- The GeoJSOn must be converted to OSM format.
- The conversion should also include the associated OSM usernames of the mapper for each geometry, if provided.

**Describe alternatives you've considered**

- Is an endpoint required at all? Can this be done via the frontend and the OSM API instead?

**Comment by manjitapandey** [comment]
@spwoodcock , Is there any chance we can make this section of verification compulsory during form or we can convince project creator to add this section while uploading custom form too, since it can be the important part on data conflation. 
![image](https://github.com/hotosm/fmtm/assets/97273021/7481d87f-21ae-4591-a2c2-f59a196426c2)

**Comment by spwoodcock** [comment]
Yes we certainly can! I can make the digitisation verification `required` in the XLSForm.

I think the only flow we should suggest to project creators is to either use our forms, or extend from our forms. We can hide rows that are important for FMTM functionality so that users don't accidentally edit them.

**Comment by spwoodcock** [comment]
In the latest form from osm-fieldwork, included in this PR https://github.com/hotosm/fmtm/pull/1555
the XLSForm now looks like this:

![image](https://github.com/hotosm/fmtm/assets/78538841/fbb6e81c-a783-4ddb-8f28-4eecc621e0c1)

With the verification fields in the essential section and hidden by default.

I just made the `digitization_correct` yes/no question compulsory for the next release too.

**Comment by manjitapandey** [comment]
Decided on injecting those fields as mandatory. Ref: #1722

**Comment by spwoodcock** [comment]
More context for this here: https://github.com/hotosm/fmtm/issues/1548#issuecomment-2265859147
